content/server-server-api.md

```diff
@@ -187,7 +187,7 @@ to send. The process overall is as follows:
     for `<hostname>`.
 
 6.  If the `/.well-known` request returned an error response, and the
-    SRV record was not found, an IP address is resolved using CNAME, AAAA and A
+    SRV records were not found, an IP address is resolved using CNAME, AAAA and A
```

Comment:
Isn't matrix set up in a way where there only is one SRV record at any given time? (Though on a quick search I didnt see anything not allowing multiple)

Reply:
Standard SRV resolution allows for multiple records, and Matrix never forbade it.  But also, steps 4 and 5 refer to different SRV records (`_matrix-fed._tcp.<hostname>` and `_matrix._tcp.<hostname>`), so it really should be plural.

On the other hand "the SRV records were not found" sounds like SRV records exist but could not be found for whatever reason.  So maybe something like "... and no SRV records are present ...", or "... and no SRV records are found ..." might be better?